drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -83,9 +94,14 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMedium1394:
                     case NdisPhysicalMediumWirelessWan:
 #endif
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+                    /* The Native 802.11 interface is supported in NDIS 6.0 and later versions.
+                     * https://learn.microsoft.com/en-us/windows-hardware/drivers/network/oid-gen-physical-medium
+                     */
```

Comment:
A simpler comment should be enough.
```suggestion
                    /* NDIS 6.0+ (Vista+) */
```

Reply:
Indeed, copying a text from MSDN isn't a best option, so short comment is actually better.

Reply:
Committed.